### PR TITLE
Refactor/diviz11/mesheryctl-filter-import-update

### LINKED ISSUE #8244: [mesheryctl] Rename --config flag in mesheryctl filter import
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
`mesheryctl filter import` has a flag `--config` with a shorthand of `-c` 
```
abrafi@AbMacAir mesheryctl % ./mesheryctl filter import -h 
Import a WASM filter from a URI (http/s) or local filesystem path

Usage:
  mesheryctl filter import [URI] [flags]

Examples:

// Import a filter file from local filesystem
mesheryctl filter import /path/to/filter.wasm

// Import a filter file from a remote URI
mesheryctl filter import https://example.com/myfilter.wasm

// Add WASM configuration 
// If the string is a valid file in the filesystem, the file is read and passed as a string. Otherwise, the string is passed as is.
// Use quotes if the string contains spaces
mesheryctl filter import /path/to/filter.wasm -config [filepath|string]

// Specify the name of the filter to be imported. Use quotes if the name contains spaces
mesheryctl filter import /path/to/filter.wasm -name [string]


Flags:
  -c, --config string   (optional) WASM configuration filepath/string
  -h, --help            help for import
  -n, --name string     (optional) filter name

Global Flags:
  -t, --token string   Path to token file default from current context
  -v, --verbose        verbose output
```

The `--config` flag for `mesheryctl filter import` overrides the global flag `--config` for `mesheryctl`

```
abrafi@AbMacAir mesheryctl % ./mesheryctl -h               
As a self-service engineering platform, Meshery enables collaborative design and operation of cloud native infrastructure.

Usage:
  mesheryctl [flags]
  mesheryctl [command]

Examples:

// Base command
mesheryctl

// Display help about command/subcommand
mesheryctl --help
mesheryctl system start --help

// For viewing verbose output
mesheryctl -v [or] --verbose


Available Commands:
  app         Cloud Native Apps Management
  completion  Output shell completion code
  filter      Cloud Native Filter Management
  help        Help about any command
  mesh        Cloud Native Lifecycle Management
  pattern     Cloud Native Patterns Management
  perf        Performance Management
  system      Meshery Lifecycle Management
  version     Version of mesheryctl

Flags:
      --config string   path to config file (default "/Users/abrafi/.meshery/config.yaml")
  -h, --help            help for mesheryctl
  -v, --verbose         verbose output

Use "mesheryctl [command] --help" for more information about a command.
```

#### Desired Behavior

Rename `--config` in `mesheryctl filter import` to `--wasm-config` having a shorthand of `-wc`

Update [mesheryctl command tracker](https://bit.ly/3dqXy1q) sheet when done.

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by diviz11**
**Notes for Reviewers**

This PR fixes #8244 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [✅] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/filter/import.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ mesheryctl filter import https://example.com/myfilter.wasm
 // Add WASM configuration 
 // If the string is a valid file in the filesystem, the file is read and passed as a string. Otherwise, the string is passed as is.
 // Use quotes if the string contains spaces
-mesheryctl filter import /path/to/filter.wasm -config [filepath|string]
+mesheryctl filter import /path/to/filter.wasm -wasm-config [filepath|string]
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/filter/import.go:50]:
```suggestion
mesheryctl filter import /path/to/filter.wasm --wasm-config [filepath|string]
```

File: mesheryctl/internal/cli/root/filter/import.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ mesheryctl filter import https://example.com/myfilter.wasm
...
 
 // Specify the name of the filter to be imported. Use quotes if the name contains spaces
 mesheryctl filter import /path/to/filter.wasm -name [string]
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/filter/import.go:53]:
```suggestion
mesheryctl filter import /path/to/filter.wasm --name [string]
```

**Review by alphaX86 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@abdullah1308 feedback?

**Review by alphaX86 — DISMISSED** [review]
Since cobra doesn't support more than 1 ASCII character for shorthand flag, I'm approving this for now

LGTM

**Review by suhail34 — APPROVED** [review]
LGTM
For supporting more than one ascii letter i think we have to create our custom parsing logic

**Review by suhail34 — CHANGES_REQUESTED** [review]

**Comment by Chadha93** [comment]
@diviz11  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Pull request merged**